docker/Dockerfile

```diff
@@ -59,14 +46,7 @@ RUN \
   # - 'ssmtp' to be able to send mails
   # - 'util-linux' not sure if this is required
   apk upgrade --no-cache && apk add --no-cache \
```

Comment:
Can you change the comments here to something like this:
`# Upgrade system and install 'ssmtp' to be able to send mails`